performance/SwiftUIPerformanceQuickstart/PerformanceExample/README.md

```diff
@@ -0,0 +1,54 @@
+Firebase Performance Monitoring Quickstart
+=============================
+
+# Firebase Performance Monitoring
+
+Firebase Performance Monitoring is a free mobile app performance analytics service. It
+provides detailed information about the performance of your apps automatically,
+but works at its best with Timers set by you.
```

Comment:
It also enables to measure performance of any code of your application.

Reply:
Done ✅